src/App.js

```diff
@@ -1,27 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
```

Comment:
- [ ] Please change the `app` function-based component to class-based component, As it is the requirement of the project that all components should be class-based.

### See the image of the requirement 

![image](https://user-images.githubusercontent.com/83803414/171435040-610a0c9d-5308-462f-9a78-f06f58e08bb2.png)